api/base/settings/__init__.py

```diff
@@ -28,14 +26,14 @@
         assert getattr(local, setting, None) and getattr(local, setting, None) != getattr(defaults, setting, None), '{} must be specified in local.py when DEV_MODE is False'.format(setting)
 
 def load_origins_whitelist():
-    global ORIGINS_WHITELIST
+    global CORS_ORIGIN_WHITELIST
     from osf.models import Institution, PreprintProvider
 
-    institution_origins = tuple(domain.lower() for domain in itertools.chain(*Institution.objects.values_list('domains', flat=True)))
+    institution_origins = tuple(f'https://{domain.lower()}' for domain in itertools.chain(*Institution.objects.values_list('domains', flat=True)))
```

Comment:
Note to myself: ah, this may break local for `http` env but it is a very rare case, in fact, we don't do domains locally at all for institutions, it's too much trouble to do a condition here